qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -35,6 +35,22 @@
                     flags = "arch_capabilities"
                     check_guest_cmd = "cat /sys/devices/system/cpu/vulnerabilities/mds | grep '%s'"
                     expect_items = 'Not affected'
+                - test_la57:
+                    # support RHEL.8 guest or later
+                    no RHEL.6 RHEL.7
+                    type = x86_cpu_flag_la57
+                    start_vm = no
+                    flags = "la57"
+                    auto_cpu_model = no
+                    check_cmd = 'grep "address sizes" /proc/cpuinfo | uniq | grep "${bits_virtual} bits virtual"'
```

Comment:
Btw, could you pls check if there is a cmd for windows guest?

Reply:
Not found yet, maybe we can use wsl function in the future to auto windows guest.